stellar-soroban-rpc/debian/stellar-soroban-rpc.postinst

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+# postinst script for stellar-soroban-rpc
+#
+# see: dh_installdeb(1)
+
+set -e
+
+# summary of how this script can be called:
+#        * <postinst> `configure' <most-recently-configured-version>
+#        * <old-postinst> `abort-upgrade' <new version>
+#        * <conflictor's-postinst> `abort-remove' `in-favour' <package>
+#          <new-version>
+#        * <postinst> `abort-remove'
+#        * <deconfigured's-postinst> `abort-deconfigure' `in-favour'
+#          <failed-install-package> <version> `removing'
+#          <conflicting-package> <version>
+# for details, see https://www.debian.org/doc/debian-policy/ or
+# the debian-policy package
+ROLE=stellar;
+DBNAME=stellar;
```

Comment:
You don't seem to be using this variable. Can it be removed?

Reply:
yes, removed - https://github.com/stellar/packages/pull/183/commits/bed3eef1b520e521cd4886c085e08dc553abdce6